<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgIn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgOut">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionXls"/>
   </widget>
   <widget class="QMenu" name="menuPre_Prosesing">
    <property name="title">
     <string>Pre Prosesing</string>
    </property>
    <addaction name="actionReduksiNoise"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionSharpening"/>
   </widget>
   <widget class="QMenu" name="menuEkstraksi_Citra">
    <property name="title">
     <string>Ekstraksi Citra</string>
    </property>
    <addaction name="actionThreshold"/>
    <addaction name="actionPlotRGB"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPre_Prosesing"/>
   <addaction name="menuEkstraksi_Citra"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReduksiNoise">
   <property name="text">
    <string>Reduksi Noise</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionSharpening">
   <property name="text">
    <string>Sharpening</string>
   </property>
  </action>
  <action name="actionXls">
   <property name="text">
    <string>Export Excel</string>
   </property>
  </action>
  <action name="actionThreshold">
   <property name="text">
    <string>Threshold</string>
   </property>
  </action>
  <action name="actionPlotRGB">
   <property name="text">
    <string>Plotting RGB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
